docs/qa/qa-test-plan.md

```diff
@@ -0,0 +1,332 @@
+
+#### Item to test #2.1: Timed Text Editor - double click on words
+
+![timed-text-editor-double-click-on-word](./images/timed-text-editor-double-click-on-word.gif)
+
+##### Steps:
+- Double click on a word
+##### Expected Results: 
+- [ ] Expect play head and current time it to jump to the corresponding point in the media
+- [ ] Expect the text before the word to be darker color 
+- [ ] Expect current word to be highlighted
+- [ ] Expect media to start playing and highlight and current word to continue
+
+#### Item to test #2.2: Timed Text Editor - click on timecodes
+
+##### Steps:
+- Click on timecode next to the text of the paragraph
+##### Expected Results: 
+-  [ ] Expect play head and current time it to jump to the corresponding point in the media beginning of paragraph 
+<!-- 
+Similar to above - expect same behaviour so perhaps not worth testing gain?
+- [ ] Expect the text before the word to be darker color 
+- [ ] Expect current word to be highlighted
+- [ ] Expect media to start playing and highlight and current word to continue
+ -->
+
+#### Item to test #2.3: Timed Text Editor - Edit speakers labels
+
+#### Steps:
+- click on speaker icon next to default speaker names
+#### Expected Results: 
+-  [ ] Expect a prompt to come up, 
+    - [ ] on add text and click on expect name to change in Timed Text Editor next to speaker label icon clicked.
+- [ ] if instead of ok click cancel, expect nothing to happen
+
+---
+
+### Item to test #3: Settings
+
+#### Item to test #3.1: Settings - show
+
+##### Steps:
+- click on settings, cog  icon top left
+##### Expected Results: 
+-  Expect settings panel to come up 
+
+#### Item to test #3.2: Settings - hide
+
+##### Steps:
+- click on settings panel cross, top right
+##### Expected Results: 
+-  Expect settings panel to hide
+
+
+#### Item to test #3.3: Settings - pause while typing - on
+
+##### Steps:
+- click settings icon top left
+- in settings panel, click toggle for "pause while typing" to be on
+- in TimedTextEditor double click on a word, media starts playing
+- start typing to correct a word
+#### #Expected Results: 
+- [ ] Expect media to pause while typing
+- [ ] Expect media to re-start a few seconds after stopped typing
+
+#### Item to test #3.4: Settings - pause while typing - off
+
+##### Steps:
+- click settings icon top left
+- in settings panel, click toggle for "pause while typing" to be off
+- in TimedTextEditor double click on a word, media starts playing
+- start typing to correct a word
+##### Expected Results: 
+- [ ] Expect media not to pause while typing, and continue playing
+
+#### Item to test #3.5: Settings - scroll sync - on
+scroll into view 
+
+##### Steps:
+- click settings icon top left
+- in settings panel, click toggle for "scroll sync" to be on 
+- in TimedTextEditor click on progress bar, further away from current playhead
+##### Expected Results: 
+- [ ] Expect text in timed text editor in view to jump to corresponding current time
```

Comment:
Does it jump only when it goes from paused to playing?

Reply:
Yes, but if in the setup steps you have clicked on the progress bar, clicking on progress bar should start playing already (and this should be tested as separate step further upstream),no?